metadb/rqlite/system_test/single_node_test.go

```diff
@@ -170,8 +170,8 @@ func Test_SingleNodeConcurrentRequests(t *testing.T) {
 	if r != `{"results":[{"columns":["COUNT(*)"],"types":[""],"values":[[200]]}]}` {
 		t.Fatalf("test received wrong result got %s", r)
 	}
-}
-
+}*/
+/*
 func Test_SingleNodeConcurrentRequestsCompressed(t *testing.T) {
```

Comment:
should use t.Skip() to skip test than comment it

Reply:
we keep it for passing staticcheck
thank you